sdk/communication/azure-communication-administration/tests/test_communication_identity_client.py

```diff
@@ -65,3 +75,6 @@ def test_delete_user(self, connection_string):
         identity_client.delete_user(user)
 
         assert user.identifier is not None
+    
+    def get_endpoint_from_connection_string(self, connection_string):
```

Comment:
inside `_shared/utils` there is a method for parsing connection string. Let's use that.